libnavui-androidauto/src/main/java/com/mapbox/androidauto/navigation/roadlabel/RoadLabelSurfaceLayer.kt

```diff
@@ -33,48 +32,40 @@ import com.mapbox.navigation.ui.shield.model.RouteShield
 @OptIn(MapboxExperimental::class)
 class RoadLabelSurfaceLayer(
     val carContext: CarContext,
-) : CarSurfaceLayer() {
```

Comment:
in order to make this have a free upgrade, it seems we need to fix this part (in a separate pull request).

Reply:
could just move this class to the internal package

Reply:
but we also want to have a public component for this.

```
CurrentRoadLabel : MapboxCarMapObserver
```

Reply:
#6546